articles/logic-apps/logic-apps-move-from-mabs.md

```diff
@@ -84,7 +84,7 @@ A bridge can take data received from a connector,
 do some work with the data, and send the results to another system. 
 Logic Apps does the same by supporting the same pipeline-based 
 interaction patterns as BizTalk Services and also providing other integration patterns. 
-The [XML Request-Reply Bridge](https://msdn.microsoft.com/library/azure/hh689781.aspx) 
+The [XML Request-Reply Bridge](/previous-versions/azure/hh689781(v=azure.100)) 
```

Comment:
Please use this URL:

https://docs.microsoft.com/previous-versions/azure/hh689781(v=azure.100)